tools/probes/wave.h

```diff
@@ -0,0 +1,45 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2019 Intel Corporation. All rights reserved.
+ *
+ * Author: Adrian Bonislawski <adrian.bonislawski@intel.com>
+ */
+
+#ifndef __WAVE_H__
+#define __WAVE_H__
+
+#define HEADER_RIFF 0x46464952
+#define HEADER_WAVE 0x45564157
+#define HEADER_FMT  0x20746d66
+#define HEADER_DATA 0x61746164
```

Comment:
I know that these are just 4-chars array converted to ints, but it may not be obvious for everyone, so you can add some note.

Reply:
added comments with ascii explanatory